playground/index.html

```diff
@@ -86,6 +114,11 @@
 
 			async function init() {
 
+				const container = document.createElement( 'div' );
+				container.className = 'splitview';
+
+				document.body.appendChild( container );
```

Comment:
We can use `this.domElement` instead of `document.body` here and in others appends? This would make the editor more flexible if we added it embed in editor for exemple. I think we can do the same thing with the classes, for example: `flow splitview` intead of just `splitview`.

Reply:
`this.domElement` is `null` in the context of the script (which is directly embedded into the html). Thus, I am not sure how to implement that, can you give me a hint?